crmsh/utils.py

```diff
@@ -42,6 +42,14 @@ def to_ascii(s):
         return s
 
 
+def completed(item, args, use='='):
+    """Check whether item has been completed in args"""
+    for arg in args:
+        if arg.startswith(item+use):
+            return True
+    return False   
+
+
```

Comment:
how about a more general function `any_startswith`:

```
def any_startswith(iterable, prefix):
    """Return first element in iterable which startswith prefix, or None."""
    for element in iterable:
        if element.startswith(prefix):
            return element
    return None
```

and then use like this:

```
return [s+'=' for s in constants.rsc_meta_attributes if utils.any_startswith(args, s + '=') is None]
```